### ISSUE TITLE
Conda environment - Documentation

### ISSUE THREAD

**Issue by pagrubel** [lanl/BEE#621]
Should we give an example of making a conda environment that one can use to install beeflow.

On darwin I did this:
module load miniconda3
create -n beeflow python=3.9
conda activate beeflow
pip install hpc-beeflow

**Comment by pagrubel** [comment]
And then to use:
conda activate beeflow

**Comment by pagrubel** [comment]
hmm that did not work but this did:

module load miniconda3
conda create -n beeflow
conda activate beeflow
conda install pip
pip install hpc-beeflow

**Comment by pagrubel** [comment]
On Crusher I was able to install in a python environment

create environment:
```
module load cray-python
mkdir crusher-env
python3 -m venv crusher-env
```

load environment and install hpc-beeflow:
```
source crusher-env/bin/activate
pip install hpc-beeflow
```

**Comment by pagrubel** [comment]
Addressed in PR #666
